src/Query.php

```diff
@@ -674,6 +670,12 @@ public function where($field, $cond = null, $value = null, $kind = 'where', $num
 
         switch ($num_args) {
             case 1:
+                if (is_string($field)) {
+                    $field = $this->expr($field);
+                }
+
+                $field->consumeWrappedInParenthesis = true;
```

Comment:
All expressions are wrapped on principle better safe than sorry.
It can be precised by checking if expression template contains and/or words but `Expression::$template` is protected

Reply:
I belive this is good, but is only in where enough?

what if we define field x = `a, b` expr, when we select it, will it be wrapped in select like `select (a, b) as x` ?

Reply:
in other cases/num_args != 1, there is no issue?